src/pages/Analytics/AllTokens.tsx

```diff
@@ -176,7 +176,13 @@ function AllTokens() {
                         margin: 0 !important;
                       `}
                     >
-                      <Col xs="content">
+                      <Col
+                        xs="content"
+                        css={css`
+                          line-height: 1;
+                          font-size: 0;
+                        `}
+                      >
                         <AssetIcon asset={{ icon: asset?.icon }} />
```

Comment:
If you use the inline-block property, there is an issue with unwanted spacing. 
Perhaps you have used line-height and font-size to address this. 
How about removing the 'display: inline-block' from the **AssetIcon** component?🤔